.travis.yml

```diff
@@ -7,7 +7,6 @@ env:
     - TOX_ENV=py33
     - TOX_ENV=py34
     - TOX_ENV=pypy
-    - TOX_ENV=pypy3
```

Comment:
Didn't—it's broken on master as well. Pip 9 doesn't support this version of pypy. ABG and I decided to drop support as there appears to be no way to configure a custom version of pip in tox. When pypy3 catches up you can bring it back.